AFQ/tractography.py

```diff
@@ -14,9 +14,9 @@
 
 
 def track(params_file, directions="det", max_angle=30., sphere=None,
-          seed_mask=None, n_seeds=1, random_seeds=False, stop_mask=None,
-          stop_threshold=0, step_size=0.5, min_length=10, max_length=1000,
-          odf_model="DTI"):
+          seed_mask=None, n_seeds=1, random_seeds=False, random_seed=None,
```

Comment:
This is a very confusing variable name, considering we also already have `random_seeds` 😄

How about `rng_seed`?